Cargo.toml

```diff
@@ -2,13 +2,13 @@
 
 name = "persistent"
 authors = ["Jonathan Reem <jonathan.reem@gmail.com>"]
-version = "0.3.0"
+version = "0.6.0"
```

Comment:
Not sure about bumping this to match iron. Other iron crates also don't necessarily match the iron version I think. So probably just bump it to `0.4.0` instead.

Not sure what @untitaker thinks.

Reply:
Hmm... I thought it should be match because router package has same version of iron.  But it's OK to fix it as `0.4.0`. How do guys think?

Reply:
I rethought `0.4.0` is better. I'm on changing it.

Reply:
`mount` just has `0.4.0`: https://github.com/iron/mount/pull/95/files